tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/MetadataProvider.java

```diff
@@ -31,9 +31,9 @@
 
   String getDatabaseName();
 
-  Collection<String> getCatalogs();
+  Collection<String> getSchemas();
 
-  Collection<String> getTables(@Nullable String catalog);
+  Collection<String> getTables(@Nullable String schemaPattern, @Nullable String tablePattern);
 
-  TableDesc getTableDescriptor(String catalogName, String tableName) throws UndefinedTablespaceException;
+  TableDesc getTableDescriptor(String schemaName, String tableName) throws UndefinedTablespaceException;
```

Comment:
TableDesc was a abbr of TableDescriptor. For consistent, I'll change it to TableDesc.